package/opt/hassbian/suites/monitor.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+function monitor-show-short-info {
+  echo "Setup for Monitor. Bluetooth-based passive presence detection."
+}
+
+function monitor-show-long-info {
+  echo "This script installs Monitor a script for Bluetooth-based passive presence detection."
+  echo "The Mosquitto suite will also be installed since it's a dependency for Monitor."
+  echo "This script also optionaly installs a service that can be enabled to run Monitor."
+}
+
+function monitor-show-copyright-info {
+  echo "Original concept by Landrash <https://github.com/landrash>."
+}
+
+function monitor-install-package {
+echo -n "Installing Mosquitto package: "
+
+echo "Checking the installation..."
+validation=$(pgrep -f mosquitto)
+if [ ! -z "${validation}" ]; then
+  echo "Existing Mosquitto install detected. Skipping install of Mosquitto."
+else
+  echo "Installing Mosquitto"
+hassbian-config install mosquitto --accept
+fi
+
+echo "Creating Monitor folder..."
+mkdir /opt/monitor || exit
+cd /opt || exit
+
+echo "Cloning Monitor git repository"
+git clone git://github.com/andrewjfreyer/monitor
+
+echo "Running interactive setup"
+cd /opt/monitor || exit
+chmod /opt/monitor/monitor.sh || exit
```

Comment:
```suggestion
chmod +x /opt/monitor/monitor.sh || exit
```
The script fails here, guessing the idea was to make it executable?

```
chmod: missing operand after ‘/opt/monitor/monitor.sh’
```